Dockerfile

```diff
@@ -60,6 +60,20 @@ RUN GO111MODULE=on go get \
 
 RUN apt-get install -yq clang-format
 
+# Minio mint test suite
+RUN apt-get install -y wget jq curl git dnsmasq python3-dateutil
+RUN git init
+RUN git remote add -f origin https://github.com/minio/minio
+RUN git config core.SparseCheckout true
+RUN echo "mint" >> .git/info/sparse-checkout
+RUN git pull --depth=1 origin master
```

Comment:
You should be able to do something like:
```
git clone https://github.com/minio/minio --no-checkout --depth=1 minio &&
cd minio &&
git sparse-checkout init --cone &&
git sparse-checkout mint
```

Reply:
This does not work with the image we're building. The sparse-checkout command is an experimental feature (https://git-scm.com/docs/git-sparse-checkout) and I think it's not available for the version of git we install (2.20.0 apparently).

Reply:
Oh well.